sdk/src/main/java/com/horizen/account/utils/Account.java

```diff
@@ -2,10 +2,13 @@
 
 import org.web3j.crypto.Keys;
 
+import java.math.BigInteger;
+
 public final class Account {
 
     public static final int ADDRESS_SIZE = Keys.ADDRESS_LENGTH_IN_HEX / 2;
     public static final int GAS_LIMIT = 30000000;
+    public static final BigInteger INITIAL_BASE_FEE = BigInteger.valueOf(1000000000);
```

Comment:
We need to choose the better place for the GAS_LIMIT and INITIAL_BASE_FEE constants.
They are related to block, not to account.
Let's create a scala object in utils package, for example.